content/en/events/kcc2021/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Contributor Celebration"
+weight: 1
+slug: "kcc2021"
+aliases: [ "/celebration", "/kcc2021" ]
+description: |
+   It's been a rough year, and without any in person events we've lost the best
+   part of the Contributor Summits - the Hallway Track. The Kubernetes Contributor
+   Celebration is an attempt to reclaim that and celebrate our accomplishments.
+   It's a time for us to relax, chat and do something fun with your fellow
+   contributors!
```

Comment:
This should be updated - It's been 2 years since the pandemic started and we did have a small gathering at kubecon NA